dpdata/vasp/outcar.py

```diff
@@ -122,7 +122,10 @@ def analyze_block(lines, ntot, nelm, ml = False):
                 cell.append([float(ss) 
                              for ss in tmp_l.replace('-',' -').split()[0:3]])
         elif viral_token[ml_index] in ii:
-            tmp_v = [float(ss) for ss in lines[idx+viral_index[ml_index]].split()[2:8]]
+            in_kB_index = viral_index[ml_index]
+            while lines[idx+in_kB_index].split()[0] != "in" :
```

Comment:
`not lines[idx+in_kB_index].split()[0:2] == ["in", "kB"] and idx+in_kB_index < len(lines)`
We should also raise an error in the case "in kB" is not found

Reply:
To avoid IndexError, I turn `idx+in_kB_index < len(lines)` to the first place.

Reply:
You are right